requirements.txt

```diff
@@ -1,37 +1,48 @@
-PyQt5
-matplotlib
-numpy
 altgraph==0.17.3
 attrs==22.1.0
 beautifulsoup4==4.11.1
 certifi==2022.6.15
 charset-normalizer==2.1.1
 colorama==0.4.5
+contourpy==1.0.7
```

Comment:
Where are these extra dependencies coming from? Do we need these?

Reply:
This is all coming from actually specifying a version of matplotlib. De we still need that?